config/odp-linux-generic.conf

```diff
@@ -168,6 +171,17 @@ timer: {
 	# inline timer polling rate in nanoseconds. By default, this defines the
 	# maximum rate a thread may poll timers. If a timer pool is created with
 	# a higher resolution than this, the polling rate is increased
-	# accordingly.
+	# accordingly. Ignored when inline timer is not used.
 	inline_poll_interval_nsec = 500000
+
+	# Inline timer use of threads
+	#
+	# Select which thread types process non-private timer pools in inline
+	# timer implementation. Thread type does not affect private timer
+	# pool procesessing, those are always processed by the thread which
+	# created the pool. Ignored when inline timer is not used.
+	#
+	# 0: Both control and worker threads process non-private timer pools
+	# 1: Only worker threads process non-private timer pools
+	inline_thread_type = 0
```

Comment:
You could also add  option e.g.:
# 2: Only control threads process non-private timer pools